### DISCUSSION TITLE
New tutorial section on Pynq

### DISCUSSION THREAD

**Discussion by thesps** [fastmachinelearning/hls4ml#493] [Announcements]
The [hls4ml tutorial](https://github.com/fastmachinelearning/hls4ml-tutorial) has a new section “part 7; deployment” on deploying your neural network on hardware, targeting a pynq-z2 board with the new `VivadoAccelerator` backend.

In the new tutorial, we show you how to take one of the neural networks trained in a previous section and create a bitfile (or "Overlay") that you can load onto a pynq-z2 device for fast inference. 

The `VivadoAccelerator` backend was released with `hls4ml` `v0.6.0` / `coris` with support for `pynq-z2` and `zcu102` boards. What other boards would you like to see supported? Let us know👇

**Comment by Fe-r-oz** [comment]
Kindly please have support for Arty A7 100T as well.

It's a low-cost medium resource board that can be used for educational purposes, so it's best for university students. More powerful than TUL PYNQ Z2 though!

**Reply by amahpour** [reply]
I'm working on an example with the Arty A7 but [need help with the HLS generated code](https://github.com/fastmachinelearning/hls4ml/discussions/1041). Any chance you can help out?

**Reply by Fe-r-oz** [reply]
My apologies for the delayed response.

I shifted my efforts to PYNQ-Z2 instead of Arty A7.

**Comment by JochiSt** [comment]
Support for the Nexys4, which is also based on the XC7A100T and the ZyBo would be great.

**Comment by phucducnguyen** [comment]
it would be great if we have support for KV260 and KR260 as well. They are also good boards
